bin/organelle_contamination_recommendation.py

```diff
@@ -10,21 +10,13 @@
 
 def main():
     parser = argparse.ArgumentParser(description="Generate text for ENA submission")
-    parser.add_argument("--input", type=str, help="List of input files (space seperated list)", default=None)
+    parser.add_argument("--input", type=str, help="contamination recommendation file", default=None)
```

Comment:
This should be something like `Input BED file` instead of `contamination recommendation file`. The script takes a BED file as its input, and a contamination removal recommendation file is its output